.github/workflows/update-codeclimate-coverage.yml

```diff
@@ -0,0 +1,27 @@
+# Summary:
+# Run Unit and coverage tests, then upload it to Code Climate dashboard
+# Read ./README.md for extensive documentation
+
+name: Update Code Climate Coverage
+
+on:
+  push:
+    branches:
+      - 'v2-mst-aptd-gcms-lcz-sty'
+
+jobs:
+  run-tests-coverage:
+    name: Run tests coverage
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@master
+        with:
+          node-version: '12.x'
+      - run: npm install -g yarn
+      - run: yarn install
+      - uses: paambaati/codeclimate-action@v2.6.0
+        env:
+          CC_TEST_REPORTER_ID: 1977a1d86aebb0b8ad0fde9af8f081c5703d7320763258db0031142a0e196d76
```

Comment:
Doc: Add note about changing this for their own integration

Can't we store this in a github env var instead? Not particularly sensitive, but would avoid people forking NRN using our own test reporter id by mistake (will happen)
Also, may be considered sensitive by some, so probably a better practice to not show it publicly.